taxonomies/ingredients.txt

```diff
@@ -45383,6 +45397,11 @@ fi:juurikaskuitu
 sv:betfiber
 #  fr:fibres-de-betterave has 43 products in 1 language @2018-10-22
 
+<en:beetroot
+en:dried beetroot
+fr:betterave déshydratée
+pl:suszony burak
```

Comment:
Can you remove the "dried beetroot" entry, and instead add "suszony" in the ingredients_processing taxonomy?